pkg/controller/protobuf/client/client.go

```diff
@@ -392,19 +392,51 @@ func (ctrlAdapter *controllerAdapter) List(ctx context.Context, resourceKind res
 			switch term.Op {
 			case resource.LabelOpEqual:
 				labelQuery.Terms = append(labelQuery.Terms, &v1alpha1.LabelTerm{
-					Key:   term.Key,
```

Comment:
I think there's a conversion in opposite direction in the `protobuf/`, also you should use new expression in conformance tests for COSI state